get-involved.php

```diff
@@ -29,9 +29,9 @@
         <div class="whole" id="sections-menu">
             <a class="button flat suggested-action" href="#funding">Funding</a>
             <a class="button flat suggested-action" href="#translations">Translations</a>
-            <a class="button flat suggested-action" href="#support">Support</a>
-            <a class="button flat suggested-action" href="#web-development">Web</a>
-            <a class="button flat suggested-action" href="#desktop-development">Desktop</a>
+            <a class="button flat suggested-action" href="#user-support">User support</a>
+            <a class="button flat suggested-action" href="#website">Website</a>
+            <a class="button flat suggested-action" href="#development">Development</a>
```

Comment:
I don't think changing "Desktop" to "Development" makes things more clear. Web development is also development.

Reply:
Yes, I'm definitely don't mean to question that :) The ambiguity is in omitting "development" in both. Being explicit with "Web development" and "Desktop development" would resolve the ambiguity, but I fear it would be too verbose.

My perception is that an unqualified "development" is generally understood as "desktop development" by default, especially in the context of an operating system. I don't think that relying on that to provide a concise (and hopefully sufficiently clear) label to the button inherently conveys the notion that we don't consider web development to be development.

That said, we can always do something else, as long as it resolves the ambiguity of the current labels. Would something like "Coding" or "Programming" be better?